jekyll/_cci2/orb-author.md

```diff
@@ -9,285 +9,281 @@ version:
+{:.no_toc}
 
+Author and add [Parameterized Executors]({{site.baseurl}}/2.0/reusing-config/#authoring-reusable-executors) to the `src/executors` directory.
 
-## Providing Usage Examples of Orbs
+Each _YAML_ file within this directory will be treated as an orb executor, with a name that matches its filename.
 
-_The `examples` stanza is available in configuration version 2.1 and later_
+View a full example from the [Orb Project Template](https://github.com/CircleCI-Public/Orb-Project-Template/tree/master/src/executors).
 
-As an author of an orb, you may wish to document examples of using it in a CircleCI configuration file, not only to provide a starting point for new users, but also to demonstrate more complicated use cases.
+##### Jobs
+{:.no_toc}
 
-When you have completed authoring an orb, and have published the orb, the orb will be published in the [Orb Registry](https://circleci.com/orbs/registry/). You will see your newly-created orb in the Orb Registry, which is shown below.
+Author and add [Parameterized Jobs]({{site.baseurl}}/2.0/reusing-config/#authoring-parameterized-jobs) to the `src/jobs` directory.
 
-![Orbs Registry image]({{ site.baseurl }}/assets/img/docs/orbs-registry.png)
+Each _YAML_ file within this directory will be treated as an orb job, with a name that matches its filename.
 
-### Simple Examples
-Below is an example orb you can use:
+Jobs can include orb commands and other steps to fully automate tasks with minimal user configuration.
+
+View the _[hello.yml](https://github.com/CircleCI-Public/Orb-Project-Template/blob/master/src/jobs/hello.yml)_ job example from the [Orb Project Template](https://github.com/CircleCI-Public/Orb-Project-Template/tree/master/src/jobs).
 
-{% raw %}
 ```yaml
-version: 2.1
-description: A foo orb
-
-commands:
-  hello:
-    description: Greet the user politely
-    parameters:
-      username:
-        type: string
-        description: A name of the user to greet
-    steps:
-      - run: "echo Hello << parameters.username >>"
+description: >
+  # What will this job do?
+  # Descriptions should be short, simple, and clear.
+
+docker:
+  - image: cimg/base:stable
+parameters:
+  greeting:
+    type: string
+    default: "Hello"
+    description: "Select a proper greeting"
+steps:
+  - greet:
+      greeting: "<< parameters.greeting >>"
 ```
-{% endraw %}
 
-If you would like, you may also supply an additional `examples` stanza in the orb like the example shown below:
+#### Scripts
+
+One of the major benefits of the orb development kit is a script inclusion feature. When using the `circleci orb pack` command (automated when using the orb development kit), you can use the value `<<include(file)>>` within your orb config code, for any key, to include the file contents directly in the orb.
+
+This is especially useful when writing complex orb commands, which might contain a lot of _bash_ code, _(although you could use python too!)_.
 
-{% raw %}
+{:.tab.scripts.Orb-Development-Kit-Packing}
```

Comment:
```suggestion
{:.tab.scripts.Orb_Development_Kit_Packing}
```
The JS for the tabs will turn `_` into whitespace.

Reply:
I never knew that! 🎉